### PR TITLE
Added the -Xshareclasses:none option tests

### PR DESCRIPTION
**Pull request by doomerxe**
Added the tests with -Xshareclasses:none option for the following tests:

JCL_Test
TestArrayCopy
cmdLineTester_jvmtitests
cmdLineTester_jvmtitests_hcr
cmdLineTester_jvmtitests_debug
jsr335tests
jit_jitt

Fixes: #3945

Signed-off-by: Jiahan Xi <doomerXe@gmail.com>

### CODE REVIEW

**Comment by doomerxe** [comment]
Hi @hangshao0 , sanity tests passed on ppc64le_linux, x86-64_mac, x86-64_windows (JDK 11). It is ready for review. Thank you!

File: test/functional/JIT_Test/playlist.xml [hunk 1]
```diff
@@ -118,6 +118,53 @@
 		</impls>
 	</test>
 
+	<test>
+		<testCaseName>jit_jitt_openj9_SCCnone</testCaseName>
```
Comment (hangshao0) [inline test/functional/JIT_Test/playlist.xml]: Can you change it to "jit_jitt_openj9_none_SCC" ? The same to other test case names.

**Comment by hangshao0** [comment]
@llxia can you please review ?

**Comment by pshipton** [comment]
jenkins test sanity all jdk8

**Comment by pshipton** [comment]
jenkins test sanity all jdk11

**Comment by pshipton** [comment]
PR testing was broken, should be fixed now by https://github.com/eclipse/openj9/pull/8010

jenkins test sanity all jdk8

**Comment by doomerxe** [comment]
Hi @pshipton , could you please re-run the tests, I just rebased it. Thank you so much!

**Comment by pshipton** [comment]
jenkins test sanity all jdk8

**Comment by pshipton** [comment]
> I just rebased it

Any particular reason?

**Comment by doomerxe** [comment]
> > I just rebased it
> 
> Any particular reason?

Oh I think I need to rebase it to get the PR testing fix in my branch. :)

**Comment by pshipton** [comment]
> I think I need to rebase it to get the PR testing fix in my branch.

Ya, good point, you probably did. I forgot about that.

**Comment by pshipton** [comment]
jenkins test sanity all jdk11

**Pull request merged**